### ISSUE TITLE
Alacritty starts in '/' on macOS after #987

### ISSUE THREAD

**Issue by ray851107** [jwilm/alacritty#1006]
Which operating system does the issue occur on?
MacOS

**Comment by chrisduerr** [comment]
I'm sorry, I wasn't able to double-check this because my VM had some OpenGL errors. I've opened another PR to revert this for macos, please let me know if this works for you: #1007

**Comment by ray851107** [comment]
Yes, it works correctly. Thank you!
